### ISSUE TITLE
Using Flask-MDE with in application factory leads to - AttributeError: 'NoneType' object has no attribute 'context_processor'

### ISSUE THREAD

**Issue by bittobennichan** [bittobennichan/Flask-MDE#2]
Using Flask-MDE with in application factory leads to the following error
```
...
Original exception was:
Traceback (most recent call last):
  File "manage.py", line 2, in <module>
    from app import create_app, db
  File "/home/bitto/Desktop/serious_projects/ctrlv/app/__init__.py", line 9, in <module>
    mde = Mde()
  File "/usr/local/lib/python3.6/dist-packages/flask_mde/models.py", line 47, in __init__
    app.context_processor(self.context_processor)
AttributeError: 'NoneType' object has no attribute 'context_processor'
```
**Steps to reproduce**
Use Flask-MDE in application factory pattern like
```
from flask import Flask
from flask_mde import Mde
mde = Mde()


def create_app():
    app = Flask(__name__)
    mde.init_app(app)
    return app
```

**Comment by bittobennichan** [comment]
I have already isolated the issue. Will be fixed in the next library update (1.1.2)

**Comment by bittobennichan** [comment]
Fixed in v1.1.2
